README.md

```diff
@@ -0,0 +1,122 @@
+# Welcome to the Bus Stop Census GitHub Page! 
+
+## Disclaimer
+Please read this from start to finish before exploring the repository. This README describes the tools needed to run the code included in this repository, the files contained in the repository, and any miscellaneous details that will help you understand what is in this repository. Ignore at your own risk. 
+
+## What is the purpose of this repository?
+The purpose of this repository is to house the data, code, figures, and products for the Bus Stop Census data anaylsis. MARTA Army has been collecting data on MARTA bus stops from February 4th, 2020 to Dec 31st, 2020. With data collection over, the task remains to clean, validate, and analyze the data. The repository documents this process and provides an organized and replicable framework for the public to use for their own analyses.
+
+## R and RStudio Set Up Instructions
+To get started, follow these instructions to install R and RStudio adapted from [Copyright (c) Software Carpentry](http://software-carpentry.org).
+
+### R and RStudio
+- Download and install [R, a free software environment for statistical computing and graphics](https://www.r-project.org/) from [CRAN](https://cran.rstudio.com/), the Comprehensive R Archive Network. It is highly recommended to install a precompiled binary distribution for your operating system – use the links up at the top of the CRAN page linked to above!
```

Comment:
Let's use only one of these links. Which one do you prefer?

Reply:
I prefer the second. I will update it